vms/platformvm/genesis/genesis.go

```diff
@@ -41,3 +62,291 @@ func Parse(genesisBytes []byte) (*Genesis, error) {
 	}
 	return gen, nil
 }
+
+// Allocation is a UTXO on the Platform Chain that exists at the chain's genesis
+type Allocation struct {
+	Locktime uint64
+	Amount   uint64
+	Address  string
+	Message  []byte
+}
+
+// Compare compares two allocations
+func (utxo Allocation) Compare(a Allocation) int {
```

Comment:
I thought I left this comment on my last review but it looks like I forgot to post it... can we update the receiver name since we renamed the struct (UTXO -> Allocation) to avoid the conflict with the existing UTXO type? Generally the Go convention is the first letter of the type, so `a` would be suitable here.

Reply:
yep I saw the comment but forgot to make the change. thx for spotting this.